source/includes/_sub_category_search_metrics.md

```diff
@@ -0,0 +1,158 @@
+## Search Metrics for Business Sub Categories
+
+#### Resource Overview
+
+```
+Available Methods:      GET
+URI Format:             /client_reports/sub_category_search_metrics/[gmaid]?[query_params]
+```
+
+#### Usage
+
+Use GET to retrieve search metric stat report for a given advertiser and business sub category id
+
+### Business Category - Campaign Table Associations
```

Comment:
I like this image, but I wouldn't want to share this level of detail outside the organization.  Luckily, we have an easy solution for this.  Our public documentation is generated from these templates and they are actually ERB templates so we can do things like [this](https://github.com/GannettDigital/api-docs/blob/master/source/includes/_campaigns_overview_report.md.erb#L62-L68).  The if block looks ugly in the internal documentation, but it gets scrubbed right out of the external documentation.  So you can link the image naturally for the github browser where most people will see it.